zerver/tests/test_message_edit.py

```diff
@@ -897,7 +897,7 @@ def set_message_editing_params(
                     "message_content_edit_limit_seconds": orjson.dumps(
                         message_content_edit_limit_seconds
                     ).decode(),
-                    "edit_topic_policy": edit_topic_policy,
+                    "edit_topic_policy": orjson.dumps(edit_topic_policy).decode(),
```

Comment:
Not sure why this worked before.

Reply:
I think that might be mostly a noop for an integer?